src/bin/sol-fbp-generator/main.c

```diff
@@ -1886,6 +1886,9 @@ print_usage(const char *program)
         "    -j  When resolving types, use the passed DESC files. If DESC is\n"
         "        a directory then all the .json files in the directory will be used.\n"
         "        Multiple -j can be passed.\n"
+        "        Note that Soletta install dir will also be looked, as well as any\n"
```

Comment:
On 21-06-2016 16:19, Caio Marcelo de Oliveira Filho wrote:
> In src/bin/sol-fbp-generator/main.c
> <https://github.com/solettaproject/soletta/pull/2184#discussion_r67934240>:
>
>> @@ -1886,6 +1886,9 @@ print_usage(const char *program)
>>          "    -j  When resolving types, use the passed DESC files. If DESC is\n"
>>          "        a directory then all the .json files in the directory will be used.\n"
>>          "        Multiple -j can be passed.\n"
>> +        "        Note that Soletta install dir will also be looked, as well as any\n"
>
> The problem with this (always looking at a predefined set of paths) is
> that if I don't want to look to Soletta install dir. For example, I'm
> using sol-fbp-generator in my host machine to generate flows targetting
> Soletta on a non-Linux system with a different set of nodes available.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/solettaproject/soletta/pull/2184/files/320bbd683fefb9d55375f839186ebecf0926e5dc#r67934240>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADB8T6dv1tso0w-WAWqVBcz0bazliPksks5qODk_gaJpZM4I5-j9>.
>

Yes, but not looking and forcing user always know where to look appears 
bad IMHO. Maybe only looking if no -j or envvar is set?